code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm

```diff
@@ -91,6 +101,24 @@
 	var/turns_since_scan = 0
 	var/obj/movement_target
 
+/mob/living/simple_mob/animal/passive/snake/init_vore()  //CHOMPEdit - is this how you do it? (stoled)
+	if(!voremob_loaded) //CHOMPAdd
+		return //CHOMPAdd
+	.=..() //CHOMPEdit
+	var/obj/belly/B = vore_selected
+	B.name = "stomach"
+	B.desc = "The snake coils its tail around you, pushing you to the ground and pinning you with its weight. It flicks its tongue at you, before pouncing onto your head, engulfing the upper half of your body with ease as it unhinges its jaw. With greedy swallows, it pulls you deeper, and deeper. The tight walls undulate rhythmically as the danger noodle rumbles contentedly at this new meal. The snake sends the last of you down with a deep swallow, hissing softly and closing its eyes as it enjoys its new meal, tucked away nicely under those beautiful, green scales."
+	B.digest_mode = DM_HOLD
+	B.item_digest_mode = IM_HOLD
+	B.mode_flags = DM_FLAG_THICKBELLY
+	B.digest_brute = 2
+	B.digest_burn = 2
+	B.digest_oxy = 1
+	B.digestchance = 0
+	B.absorbchance = 0
+	B.escapechance = 25
+	B.escape_stun = 5
+
 /mob/living/simple_mob/animal/passive/snake/python/noodle/Life()
```

Comment:
```suggestion
// CHOMPEdit Start
/mob/living/simple_mob/animal/passive/snake/init_vore()
	if(!voremob_loaded)
		return
	.=..()
	var/obj/belly/B = vore_selected
	B.name = "stomach"
	B.desc = "The snake coils its tail around you, pushing you to the ground and pinning you with its weight. It flicks its tongue at you, before pouncing onto your head, engulfing the upper half of your body with ease as it unhinges its jaw. With greedy swallows, it pulls you deeper, and deeper. The tight walls undulate rhythmically as the danger noodle rumbles contentedly at this new meal. The snake sends the last of you down with a deep swallow, hissing softly and closing its eyes as it enjoys its new meal, tucked away nicely under those beautiful, green scales."
	B.digest_mode = DM_HOLD
	B.item_digest_mode = IM_HOLD
	B.mode_flags = DM_FLAG_THICKBELLY
	B.digest_brute = 2
	B.digest_burn = 2
	B.digest_oxy = 1
	B.digestchance = 0
	B.absorbchance = 0
	B.escapechance = 25
	B.escape_stun = 5
// CHOMPEdit End

/mob/living/simple_mob/animal/passive/snake/python/noodle/Life()
```

Reply:
Bigger sections of code changes that are not in modular_chomp have to be marked with CHOMPEdit Start/End sections.